### PR TITLE
Translation of SPV_INTEL_media_block_io from SPIRV to LLVM

### PR DESCRIPTION
**Pull request by aratajew**

### CODE REVIEW

File: lib/SPIRV/SPIRVReader.cpp [hunk 1]
```diff
@@ -1855,7 +1855,8 @@ Value *SPIRVToLLVM::transValueWithoutDecoration(SPIRVValue *BV, Function *F,
     } else if ((OCLSPIRVBuiltinMap::rfind(OC, nullptr) ||
                 isSubgroupAvcINTELInstructionOpCode(OC) ||
                 isIntelSubgroupOpCode(OC)) &&
-               !isAtomicOpCode(OC) && !isGroupOpCode(OC) && !isPipeOpCode(OC)) {
+               !isAtomicOpCode(OC) && !isGroupOpCode(OC) && !isPipeOpCode(OC) &&
+               !isMediaBlockINTELOpcode(OC)) {
```
Comment (AlexeySotkin) [inline lib/SPIRV/SPIRVReader.cpp]: Should we factor it out to some helper function?

Comment (aratajew) [reply lib/SPIRV/SPIRVReader.cpp]: Sure, I'll try to factor it out later this day.

Comment (aratajew) [reply lib/SPIRV/SPIRVReader.cpp]: Done

Comment (AlexeySotkin) [reply lib/SPIRV/SPIRVReader.cpp]: Please do the same in #381

**Pull request merged**